src/app/app.component.css

```diff
@@ -2,3 +2,7 @@
   padding: 30px;
   min-height: 87.9vh;
 }
+
+::ng-deep .mat-tooltip {
+  font-size: 16px;
+}
```

Comment:
Should you be changing the style of all mat tooltips? Other tooltips throughout the application have unintentionally been forced to follow this font size. ng-deep is dangerous in this way. Perhaps a better solution would be to use the ``matTooltipClass`` property on the component where ``[matToolTip]`` is applied and define a custom class for the unique style.

Reply:
Additionally, should you be using 1rem instead of 16px to account for user's custom font sizes in their browser?